docs/source/FAQ/team_management_migration.rst

```diff
@@ -0,0 +1,87 @@
+================================
+Teams User Management Migration
+================================
+
+Summary
+=======
+
+This is an FAQ about migrating your code to the :class:`~citrine.resources.team.Team` version of the Citrine Platform.
+
+What's new?
+====================
+
+Teams
+------
+
+The biggest change is the introduction of the Teams concept for access control. After this update, all projects and assets are owned by Teams that include one or more users on the platform. This concept was introduced to make it easier for Citrine Platform users to securely share assets across projects.
```

Comment:
"are owned by teams" --> "will be owned by Teams"

Also, I don't know if we have an .rst or Markdown style guide somewhere, but we should keep line lengths under, I don't know, does 120 characters sound reasonable? 100? Something in that rough ballpark.

Reply:
I haven't seen a style guide, but you're right, at minimum we should impose line lengths. I'll look through our other RST docs to see what the most common one is (since I think it's kind of a free-for-all). And maybe I'll cut a ticket to bring the others in line.

Reply:
In most of our docs, the lines have no cap. And doing some Googling, I see many sources stating that for maximum reader comfort, you should definitely limit line length to less than 100 (although even 100 is considered high by many sources).

So I'm going to use 100 as a guideline for these changes, and make a ticket to rein in the rest of the docs in the future.